src/content/api/compilation-hooks.mdx

```diff
@@ -471,41 +473,43 @@ compilation.hooks.processAssets.tap(
 );
 ```
 
-In addition to `name` and `stage`, you can pass a `additionalAssets` <Badge text='5.8.0+' /> option which accepts either a value of `true` or a function with `assets` as parameter:
-
-1. `true` - Run callback against assets added later by plugins.
-
-   ```js
-   compilation.hooks.processAssets.tap(
-     {
-       name: 'MyPlugin',
-       stage: Compilation.PROCESS_ASSETS_STAGE_DEV_TOOLING,
-       additionalAssets: true,
-     },
-     (assets) => {
-       // this callback will run against assets added later by plugins.
-     }
-   );
+In addition to `name` and `stage`, you can pass an `additionalAssets` <Badge text='5.8.0+' /> option which accepts either a value of `true` or a callback function that receives `assets` as a first argument:
+
+1. `true` - run the provided callback again for assets added later by plugins.
+
+In this mode, the callback will be called multiple times: once for assets added prior to the specified stage, and additional times for assets added by plugins later (on this or next stages).
+
+```js
+compilation.hooks.processAssets.tap(
+  {
+    name: 'MyPlugin',
+    stage: Compilation.PROCESS_ASSETS_STAGE_DEV_TOOLING,
+    additionalAssets: true,
+  },
+  (assets) => {
+    // process the assets
+  }
+);
```

Comment:
Seems I'm not able to edit the files from your fork so I'll leave comments.

Let's indent this. Here's how it looks like with indentation:

![image](https://user-images.githubusercontent.com/1091472/113875676-981acf00-97e9-11eb-9a6c-e519d6743022.png)

You can compare it with the current one https://webpack-js-org-git-fork-slavafomin-process-assets-webpack-docs.vercel.app/api/compilation-hooks/#processassets